issue templates im github repos hinterlegen (#2107)
redaxo/redaxo: .github/ISSUE_TEMPLATE/---bug-report.md

---

> name: "\U0001F41E Report errors or problems"

Reviewer: ```suggestion
name: "\U0001F41E Report an error or problem"
```

> about: Before you start, check if an related issue exists (https://github.com/redaxo/redaxo/issues)

Reviewer: Geht hier markdown?

```suggestion
about: Before you start, check if an [related issue exists](https://github.com/redaxo/redaxo/issues)
```

Author: Habe es versucht, leider nein

Reviewer: Ok danke.

```suggestion
about: Before you start, check if an related issue exists. See https://github.com/redaxo/redaxo/issues
```

> about: Before you start, check if a related issue exists. See https://github.com/redaxo/redaxo/issues

Reviewer: ```suggestion
about: Before you start, check whether a similar issue was already reported. Search via https://github.com/redaxo/redaxo/issues
```

> about: Before you start, check if an related issue exists. See https://github.com/redaxo/redaxo/issues

Reviewer: ```suggestion
about: Before you start, check if a related issue exists. See https://github.com/redaxo/redaxo/issues
```

---

**Description / Beschreibung**

> **Affected Versions / Verwendete Versionen**

Reviewer: ```suggestion
**Affected versions / Verwendete Versionen**
```

> **Affected versions / Verwendete Versionen**

Reviewer 2: Hier könnte man zukünftig eventuell auf den Systemreport hinweisen, mit dem man die Infos am einfachsten angeben kann.

Author: Haben wir notiert - war auch in meinem Ausgangszext drin

REDAXO:
PHP:
Browser:

> System AddOns:

Reviewer: ```suggestion
AddOns:
```

---

> name: "\U0001F41E Bug report"

Reviewer: ```suggestion
name: "\U0001F41E Report errors or problems"
```

> about: Report errors and problems. Please, before you start, check if an related issue

Reviewer: ```suggestion
about: Before you start, check if an related issue
```
